geoext.ux/ux/SimplePrint/lib/GeoExt.ux/SimplePrint.js

```diff
@@ -472,16 +477,12 @@ GeoExt.ux.SimplePrint = Ext.extend(Ext.form.FormPanel, {
                         self.printProvider.download(
                             self.printProvider.getDownloadURL(job)
                         );
-                        self.progressPanel.remove(statusComponent);
-                        statusComponent.destroy();
-                        statusComponent = null;
-                        self.progressPanel.doLayout();
-                        self.progressPanel.setHeight(35 * self.progressPanel.items.length);
-                        self.doLayout();
                         return false;
                     };
-                }
-                else {
+                    self.printProvider.download(
+                        self.printProvider.getDownloadURL(job)
+                    );
```

Comment:
I am totally ok with removing the download link if the download is started automatically. I just thought that you had a good reason to show the download link in the first place.